# GetMclPose.srv
geometry_msgs/PoseStamped GetMclPose
---
geometry_msgs/PoseStamped mcl_pose
